### ISSUE TITLE
Occasionally, somewhat quick exiting can result in failure to properly close and then silentfail to bind to the port

### ISSUE THREAD

**Issue by erwynne** [superzanti/ServerSync#316]
#### Serversync Version: 
Latest Serverside
#### Minecraft Version: 
Latest
#### Issue: 
An oddity, on Windows if you repeatedly close and open the serversync program eventually it will silently fail to bind because sometimes it will continue to run in the background without having a window open (until you, presumably, force end in task mgr). I was at my wits end to why it wasnt working even though port was fwded this morning and restarted my PC, as it was restarting, it showed up on the screen that a blank, nameless, iconless application had not closed and needed to close. Upon restarting, the issue was resolved. But it would not let go of the port, and it was not visible in the taskbar, and had been closed with the X button, and when I would run the Serversync Serverside, it would not report that the port was occupied, instead gave the all OK.

It can be resolved by restarting PC.

**Comment by rheimus** [comment]
Indeed, I have noticed in the past that SS does not always shut down gracefully.

Have not investigated it far enough to figure out if this is the software itself or if its just windows being windows 🤷, presumably its something SS is doing.

**Comment by rheimus** [comment]
One small fix could be to give SS a proper process name so its easier to terminate. At the moment I think it just shows up as `java` or something equally unhelpful.

**Comment by rheimus** [comment]
possibly related, closing the `exe` versions via the UI `X` seems to fail to shut down.
